app/models/effective/datatables/family_data_table.rb

```diff
@@ -74,6 +76,15 @@ def terminate_enrollment_type(family, allow)
       end
 
       def nested_filter_definition
+        families_tab = [
+          {scope: 'all', label: 'All'},
+          {scope: 'by_enrollment_shop_market', label: 'Employer Sponsored Coverage Enrolled', subfilter: :employer_options},
+          {scope: 'non_enrolled', label: 'Non Enrolled'}
+        ]
+        if Settings.aca.individual_market.is_active         
```

Comment:
This is good - just use the separate concern. If this is an instance method you'll need to define a method in the included block that does the same test against Settings.